web/static/help/Messaging.md

```diff
@@ -1 +0,0 @@
-../../../doc/help/Messaging.md
```

Comment:
Sorry, yup I think remove it.  I meant i'll be duplicated in the mattermost/docs repo and in the docs dir for now.  but in web/static/help there should only be Message_en.md and _es.md  (I think!!).  Does that make sense?